SLHCUpgradeSimulations/Configuration/python/fixMissingUpgradeGTPayloads.py

```diff
@@ -23,14 +23,14 @@ def fixDTAlignmentConditions(process):
     process.GlobalTag.toGet.extend( cms.VPSet(
             cms.PSet(record = cms.string("DTAlignmentErrorExtendedRcd"),
                      tag = cms.string("MuonDTAPEObjectsExtended_v0_mc"),
-                     connect = cms.string("frontier://FrontierProd/CMS_COND_ALIGN_000")
+                     connect = cms.string("frontier://FrontierProd/CMS_CONDITIONS")
```

Comment:
this could be dropped: the tag is already in use in all MC GT:
````
80X_mcRun1_design_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun1_HeavyIon_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun1_pA_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun1_realistic_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun2_asymptotic_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun2_design_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun2_HeavyIon_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
80X_mcRun2_startup_Queue	Queue	DTAlignmentErrorExtendedRcd	-	MuonDTAPEObjectsExtended_v0_mc
````